data/net/minecraft/world/level/EntityGetter.mapping

```diff
@@ -2,26 +2,39 @@ CLASS net/minecraft/world/level/EntityGetter
 	METHOD getEntities (Lnet/minecraft/world/entity/Entity;Lnet/minecraft/world/phys/AABB;)Ljava/util/List;
 		COMMENT Will get all entities within the specified AABB excluding the one passed into it. Args: entityToExclude, aabb
 		ARG 1 entity
-		ARG 2 bb
+		ARG 2 area
 	METHOD getEntities (Lnet/minecraft/world/entity/Entity;Lnet/minecraft/world/phys/AABB;Ljava/util/function/Predicate;)Ljava/util/List;
 		COMMENT Gets all entities within the specified AABB excluding the one passed into it.
 		ARG 1 entity
 		ARG 2 boundingBox
 		ARG 3 predicate
+	METHOD getEntities (Lnet/minecraft/world/level/entity/EntityTypeTest;Lnet/minecraft/world/phys/AABB;Ljava/util/function/Predicate;)Ljava/util/List;
+		ARG 1 entityTypeTest
+		ARG 2 aabb
```

Comment:
This is called `area` elsewhere in the class but `aabb` here which is both a non-capitalized acronym (bad) and annoying, why? If not `area` I'd suggest `boundingBox`